src/daemon/dlt-daemon.c

```diff
@@ -952,6 +952,12 @@ int main(int argc, char *argv[])
 
     /* --- Daemon connection init end */
 
+    /*
+     * Load dlt-runtime.cfg if available.
+     * This must be loaded before offline setup
+     */
+    dlt_daemon_configuration_load(&daemon, daemon.runtime_configuration, daemon_local.flags.vflag);
```

Comment:
Hello @kwaczor ,
Sorry for my very slow in response since I was off due to personal stuffs.

You are right that the dlt_daemon_configuration_load() must be called before dlt_daemon_local_init_p2().
But daemon.runtime_configuration will be updated inside dlt_daemon_init_runtime_configuration() so please update your PR accordingly.
After that, I think your PR are ready to be merged!

Thank you for your patience!